presto-docs/src/main/sphinx/release/release-342.md

```diff
@@ -0,0 +1,49 @@
+# Release 342 (XX Sep 2020)
+
+## General Changes
+
+* Improve performance of queries that use decimal type. ({issue}`4886`)
+* Improve query performance by adding support for dynamic filtering and dynamic
+  partition pruning to semi-join relational operator. ({issue}`5017`)
+* Improve dynamic partition pruning and query performance by reducing latency of dynamic filters collection. ({issue}`4988`)
```

Comment:
```
Reduce dynamic partition pruning overhead?
```

Reply:
i think "latency" is an important aspect 

but, this could be expressed more from end-user perspective

"Improve performance and latency of queries leveraging dynamic partition pruning with dynamic filters."

(i know it's repetitive but i think it's important to stick "dynamic filters" here, as this is the recognized feature name)